source/declarations.tex

```diff
@@ -2283,7 +2297,7 @@
 \begin{bnf}
 \nontermdef{init-declarator}\br
     declarator \opt{initializer}\br
```

Comment:
`\opt` needs to be removed here

Reply:
good catch, i totally missed that edit.   Done.